lib/Dialect/FIRRTL/Transforms/ExtractInstances.cpp

```diff
@@ -199,8 +209,16 @@ void ExtractInstancesPass::runOnOperation() {
   if (anyFailures)
     return signalPassFailure();
 
+  ClassOp sifiveMetadata;
+  for (ClassOp classOp : circtOp.getOps<ClassOp>()) {
+    if (classOp.getSymNameAttr().getValue().compare("SiFive_Metadata"))
```

Comment:
`compare` returns non-zero when they are different so probably this is not intended. Could you use symbol table look up instead of manual string comparison?

Reply:
Thanks Hideto, The `compare` was used as intended here, but using the symboltable makes sense. Updated to use it.